### PR TITLE
Converted to property is_anonymous

### PR DESCRIPTION
**Pull request by iSevenDays**
flask 0.3.0 has breaking change

Please see
https://github.com/hack4impact/flask-base/issues/124

### CODE REVIEW

File: app/account/views.py [hunk 1]
```diff
@@ -255,7 +255,7 @@ def join_from_invite(user_id, token):
     return redirect(url_for('main.index'))
 
 
-@account.before_app_request
+@account.before_app_requestis
```
Comment (abhisuri97) [inline app/account/views.py]: `before_app_requestis`. I am not aware of this syntax. is it a typo?

Comment (iSevenDays) [reply app/account/views.py]: Yes, it's a typo

**Comment by abhisuri97** [comment]
lgtm. merged :)

**Comment by iSevenDays** [comment]
Thank you!

**Pull request merged**